frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -155,6 +156,18 @@ export const EmbedFrame = ({
   const shouldApplyParameterPanelThemeChangeTransition =
     !isParameterPanelStickyStateChanging && isParameterPanelSticky;
 
+  const valuePopulatedParameters = getValuePopulatedParameters({
+    parameters,
+    values: _.isEmpty(draftParameterValues)
+      ? parameterValues
+      : draftParameterValues,
+  });
+
+  useSyncUrlParameters({
```

Comment:
So previously `ParametersListComponent` will get its `parameters` prop from 
```ts
getValuePopulatedParameters({
  parameters,
  values: _.isEmpty(draftParameterValues)
    ? parameterValues
    : draftParameterValues,
});
```

This preserves the same logic. It's the same side-effect.